client/src/telethon/_impl/client/events/queries.py

```diff
@@ -69,6 +70,21 @@ async def answer(
             )
         )
 
+    async def get_message(self) -> Optional[Message]:
+        """
+        Get the :class:`~telethon.types.Message` containing the button that was clicked.
+
+        If the message is too old and is no longer accessible, :data:`None` is returned instead.
+        """
+        peer_id_ = peer_id(self._raw.peer)
+        peer = self._chat_map.get(peer_id_, None)
+        if not peer:
+            peer = await self._client._resolve_to_packed(peer_id_)
+
+        message = (await self._client.get_messages_with_ids(chat=peer, message_ids=[self._raw.msg_id]))[0]
```

Comment:
This probably won't work. I looked in the TL file. There's `inputMessageCallbackQuery` which we probably must use instead. Probably the same thing done in `client/src/telethon/_impl/client/types/message.py` should be done here but with that type instead.

Reply:
It did indeed work for user, chat, channel, supergroup (as mentioned in PR)

Reply:
implemented